site/en/blog/related-website-sets/index.md

```diff
@@ -0,0 +1,58 @@
+description: >
+  Related Website Sets (RWS) is the new name for First-Party Sets and also brings increased flexibility in defining sets.
+date: 2023-08-31
+thumbnail: 'image/80mq7dk16vVEg8BBhsVe42n6zn82/s3iDQJUgLZV25YbtYxs1.png'
+alt: Privacy Sandbox logo
+tags: 
+  - privacy
+authors:
+  - helencho
+---
+
+Many Privacy Sandbox APIs are ramping up to General Availability (GA) in Chrome Stable in preparation for third-party cookie deprecation beginning in 2024. Some of these APIs will help preserve crucial cross-site cookie use cases, like [CHIPS](/en/docs/privacy-sandbox/chips/), and the API currently known as [First-Party Sets (FPS)](/en/docs/privacy-sandbox/first-party-sets/). In this post, we introduce Related Website Sets (RWS) — our new name for FPS that better reflects its purpose— provide a refresher on key use cases along with an update on the associated subset domain limit. 
+
+
+## Preserving critical user journeys
+
+RWS is designed to minimize disruptions to specific user-facing features once Chrome starts limiting access to third-party cookies by default. Our goal is to allow users to browse the web with minimal disruption while still upholding the privacy goals of the Privacy Sandbox. To strike this balance, RWS targets specific use cases related to website functionality:
+
+* The [ccTLD use case](https://github.com/WICG/first-party-sets/tree/main#defining-a-set-through-use-case-based-subsets) addresses breakages like the [login example](https://issuetracker.google.com/268390722) filed in our public tracker. Such cases are often addressed in the ecosystem through heuristic-based exceptions (See [ref 1](#ref-1)).
+* The [service domain use case](https://github.com/WICG/first-party-sets/tree/main#defining-a-set-through-use-case-based-subsets) addresses a common developer practice to isolate sensitive functions (like supporting an authentication flow) from user-facing domains. Such cases may be addressed in the ecosystem through targeted [exceptions](https://searchfox.org/mozilla-central/rev/3002762e41363de8ee9ca80196d55e79651bcb6b/browser/extensions/webcompat/data/shims.js#686) (See [ref 2](#ref-2)).
+*   The [associated domain use case](https://github.com/WICG/first-party-sets/tree/main#defining-a-set-through-use-case-based-subsets) provides more flexibility for the types of domains that may require third-party cookie access for critical user journeys (See [ref 3](#ref-3)). While the ccTLD and service domain use cases employ strict technical checks based on domain characteristics to minimize abuse, the associated domain uses a numeric limit. Read more about this below.
+
+
+## Associated domain limit increased to five domains 
+
+Chrome previously proposed a numeric limit of 3 domains for the Associated Subset (plus one primary domain), in alignment with our objective to prevent widespread tracking abuse. We have heard feedback from web standards participants that the limit was too low for different types of use cases. 
```

Comment:
```suggestion
Chrome previously proposed a numeric limit of three domains for the Associated Subset (plus one primary domain), in alignment with our objective to prevent widespread tracking abuse. We have heard feedback from web standards participants that the limit was too low for different types of use cases. 
```